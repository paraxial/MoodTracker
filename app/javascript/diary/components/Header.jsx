import React from 'react';

const Header = () => (
  <div className="header-bar">
    <div className="title-box">
      Today's mood
      <div className="subtitle">
        A Symptom Tracker
      </div>
    </div>
    <div className="navigation-bar">
      <span className="navigation-option">Placeholder</span>
      <span className="navigation-option">Placeholder</span>
    </div>
  </div>
);

export default Header;
