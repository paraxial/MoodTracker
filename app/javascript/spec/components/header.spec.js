import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../diary/components/Header';

describe('Header', () => {
  test('should match snapshot', () => {
    const renderer = new ShallowRenderer();
    const page = renderer
      .render(
        <Header>
          <div>Some child component</div>
        </Header>
      );

    expect(page).toMatchSnapshot();
  });
});
