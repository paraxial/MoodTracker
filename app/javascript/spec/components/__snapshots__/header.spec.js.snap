// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div
  className="header-bar"
>
  <a
    href="/"
  >
    <div
      className="title-box"
    >
      Today's mood
      <div
        className="subtitle"
      >
        A Symptom Tracker
      </div>
    </div>
  </a>
  <div
    className="navigation-bar"
  >
    <div>
      Some child component
    </div>
  </div>
</div>
`;
